### DISCUSSION TITLE
I cannot make a donation for your excellent work !

### DISCUSSION THREAD

**Discussion by denis-beurive** [Camelcade/Perl5-IDEA#2534] [General]
Hello,

I've tried to make a donation for you really useful Perl Plugin, but it is not possible.

The URL "http://www.evstigneev.com/" is unreachable.

The email address "hurricup@evstigneev.com" is unreachable.

If you need help, let us know.

Best regards,

Denis

**Comment by hurricup** [comment]
Yes, this is not possible at the moment because of world situation. I'll get back when and if it's going to be possible. Now just enjoy :)

**Reply by hoxi** [reply]
So one day after the CLion 2023.2 release, you push out a new release of Perl5-IDEA *with* Perl 5.38.0 support! Is there any way to get a donation to you now?

**Reply by hurricup** [reply]
Nope :) don't worry about this

**Comment by hurricup** [comment]
You can do this now with sponsor link i presume.
